platforms/android/build-tests/test_gradle.sh

```diff
@@ -35,7 +35,7 @@ echo "ndk.dir=${ANDROID_NDK}" > "test-gradle/samples/local.properties"
 echo "cmake.dir=$(dirname $(dirname $(which cmake)))" >> "test-gradle/samples/local.properties"
 
 echo "Run gradle ..."
-(cd "test-gradle/samples"; ./gradlew -i assemble)
```

Comment:
It is better to use dedicated variable for debugging and do not break clean user builds: https://github.com/opencv/opencv/blob/4.9.0/cmake/android/android_gradle_projects.cmake#L185

Reply:
Done.